cvat/apps/engine/views.py

```diff
@@ -398,51 +386,52 @@ def perform_create(self, serializer, **kwargs):
         # Required for the extra summary information added in the queryset
         serializer.instance = self.get_queryset().get(pk=serializer.instance.pk)
 
-    def get_export_callback(self, save_images: bool) -> Callable:
-        return dm.views.export_project_as_dataset if save_images else dm.views.export_project_annotations
-
     @extend_schema(methods=['GET'], summary='Export a project as a dataset / Check dataset import status',
         description=textwrap.dedent("""
-            To check the status of the process of importing a project dataset from a file:
-
-            After initiating the dataset upload, you will receive an rq_id parameter.
-            Make sure to include this parameter as a query parameter in your subsequent
-            GET /api/projects/id/dataset requests to track the status of the dataset import.
-            Also you should specify action parameter: action=import_status.
+            Utilizing this endpoint:
+            - to export project dataset in a specific format
+            - to check the status of the process of importing a project dataset from a file
+            is deprecated.
```

Comment:
This doesn't seem like a correct description. Checking the import status is deprecated, but initiating an export is _removed_, right?